core/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -839,7 +839,7 @@ describe("IModelTransformer", () => {
   });
 
   // WIP: Included as skipped until test file management strategy can be refined.
```

Comment:
@MichaelBelousov not sure what file management strategy is involved here but do we intend to get this test running?
Currently tries to open d:\data\bim\MergeTest\Campus.bim.

Reply:
this is a very old test that Shaun had the test data for and I don't. I vote we delete, it doesn't seem to assert anything about the result anyway.

Reply:
the other skipped transformer test should be redundant with other tests too and I think should also be deleted.